types/index.esm.d.ts

```diff
@@ -1654,7 +1654,7 @@ export interface FontSpec {
   lineHeight: number | string;
 }
 
-export type TextAlign = 'left' | 'center' | 'right';
+export type TextAlign = 'left' | 'center' | 'right' | 'inner';
```

Comment:
Old one was correct, you only changed the alignment for the ticks in the axis, this type is used for a lot of other parts where it does not apply

```suggestion
export type TextAlign = 'left' | 'center' | 'right';
```

Reply:
Done